src/lib.rs

```diff
@@ -76,15 +77,6 @@ use std::path::{PathBuf, Path};
 use std::process::{Command, Output};
 use std::str;
 
-pub fn target_supported() -> bool {
```

Comment:
It's kind of ancient history now, but I was surprised to learn that this was removed -- a breaking change.
(cuviper/elfutils-rs#1)

Reply:
Sorry! I haven't noticed that it was a public interface.

Reply:
No worries, I just wanted to connect the dots.

Reply:
Sorry for missing that back then